node/pkg/governor/mainnet_chains.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/wormhole-foundation/wormhole/sdk/vaa"
 )
 
-func chainList() []chainConfigEntry {
+func ChainList() []chainConfigEntry {
```

Comment:
Does this need to be made public? It is only referenced inside of this package.

Reply:
You're right, thanks. I've reverted the change.